frontend/src/app/_models/globals.ts

```diff
@@ -0,0 +1,88 @@
+import { Injectable } from "@angular/core";
+import notify from 'devextreme/ui/notify';
+import { DatePipe } from "@angular/common";
+
+
+@Injectable()
+export class Globals {
+
+  public toastTimeDuration: number = 1000;
+  public toastSuccessType: string = "success";
+  public toastErrorType: string = "error";
+  public companyName: string = "S4";
+  public imageProfileURLDir: string = 'https://portal.s4.co.za/gallery/images/staff/';
+  public formNavString: string = '/form';
+  public hereMapAppID = 'FWQJ1YZPbezEhJJWrywq';
+  public hereMapAppCode = '0Sow3XVx8L5B5iMdGiYUXA';
+  public emailServerDirectory = 'http://localhost:8181/emailapi/'
```

Comment:
Same story with the host thing. When its broken down and separated into a configuration it becomes easier to change out your system between local, test and production.